packages/admin-next/dashboard/src/routes/orders/order-create-claim/components/claim-create-form/claim-inbound-item.tsx

```diff
@@ -0,0 +1,247 @@
+              amount={previewItem.return_requested_total}
+            />
+          </div>
+
+          <ActionMenu
+            groups={[
+              {
+                actions: [
+                  !showReturnReason && {
+                    label: t("actions.addReason"),
+                    onClick: () =>
+                      form.setValue(`inbound_items.${index}.reason_id`, ""),
+                    icon: <ChatBubble />,
+                  },
+                  !showNote && {
+                    label: t("actions.addNote"),
+                    onClick: () =>
+                      form.setValue(`inbound_items.${index}.note`, ""),
+                    icon: <DocumentText />,
+                  },
+                  {
+                    label: t("actions.remove"),
+                    onClick: onRemove,
+                    icon: <XCircle />,
+                  },
+                ].filter(Boolean),
+              },
+            ]}
+          />
+        </div>
+      </div>
+      <>
+        {/*REASON*/}
+        {showReturnReason && (
+          <div className="grid grid-cols-1 gap-2 p-3 md:grid-cols-2">
+            <div>
+              <Form.Label>{t("orders.returns.reason")}</Form.Label>
+              <Form.Hint className="!mt-1">
+                {t("orders.returns.reasonHint")}
+              </Form.Hint>
+            </div>
+
+            <div className="flex items-center gap-1">
+              <div className="flex-grow">
+                <Form.Field
+                  control={form.control}
+                  name={`inbound_items.${index}.reason_id`}
+                  render={({ field: { ref, value, onChange, ...field } }) => {
+                    return (
+                      <Form.Item>
+                        <Form.Control>
+                          <Combobox
+                            className="bg-ui-bg-field-component hover:bg-ui-bg-field-component-hover"
+                            value={value}
+                            onChange={(v) => {
+                              onUpdate({ reason_id: v })
+                              onChange(v)
+                            }}
+                            {...field}
+                            options={return_reasons.map((reason) => ({
+                              label: reason.label,
+                              value: reason.id,
+                            }))}
+                          />
+                        </Form.Control>
+                        <Form.ErrorMessage />
+                      </Form.Item>
+                    )
+                  }}
+                />
+              </div>
+              <IconButton
+                type="button"
+                className="flex-shrink"
+                variant="transparent"
+                onClick={() => {
+                  onUpdate({ reason_id: null }) // TODO BE: we should be able to set to unset reason here
+                  form.setValue(`inbound_items.${index}.reason_id`, "")
```

Comment:
nit:
```suggestion
                  form.setValue(`inbound_items.${index}.reason_id`, null)
```

Reply:
I'll keep this here as is, its changed in my current PR